templates/opps-list-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template is used to display the list of flexible volunteer opportunities
+ * for the widget.
+ *
+ * To adjust this template copy it into your current theme within a folder called "wivm".
+ */
+
+// Global variables that are set in widget/class-widget.php via the widget settings in admin 
```

Comment:
I assume you added these globals so we can use a template. I wonder if there is any way to do this without globals. I'm sure you looked into it, so I'd love to hear what you think.